news/2022-05-24-triangles.md

```diff
@@ -0,0 +1,129 @@
+
+### Mapper panel
+
+- **Runner-up:** 3 months of osu!supporter 
+- **Winner:**  6 months of osu!supporter + *Mappers' Pick* profile badge
+
+![Mappers' Pick profile badge](/wiki/shared/news/2022-05-24-triangles/mappers@2x.png)
+
+### Storyboarder panel
+
+- **Runner-up:** 3 months of osu!supporter 
+- **Winner:**  6 months of osu!supporter + *Storyboarders' Pick* profile badge
+
+![Storyboarders' Pick profile badge](/wiki/shared/news/2022-05-24-triangles/storyboarders@2x.png)
+
+### Overall prizes
+
+This category is based on how members of both Mapper and Storyboarder panels evaluate the full beatmap package.
+
+- **Runner-up:**
+  - 6 months of osu!supporter
+  - *Overall runner-up* profile badge
+- **Winner:**
+  - 12 months of osu!supporter
+  - *Overall winner* profile badge
+  - *Elite Mapper*/*Elite Storyboarder* user titles
+  - Submission bundled with new lazer installations (after reaching Ranked status)
+
+![Overall runner-up profile badge](/wiki/shared/news/2022-05-24-triangles/overallrunnerup@2x.png)
+
+![Overall winner profile badge](/wiki/shared/news/2022-05-24-triangles/overallwinner@2x.png)
+
+### Community vote
+
+Top beatmaps will be published for a **community vote** and featured in **5 lazer playlists** (one per difficulty level). At this time, teams may reveal their beatmaps to the public.
+
+To participate in the community vote, you will need to play through any of the relevant playlists on lazer. This ensures anyone voting has actually considered all of the options!
+
+- **Community vote runner-up:** 3 months of osu!supporter
+- **Community vote winner:** 6 months of osu!supporter + *Community Pick* profile badge
+
+![Community pick profile badge](/wiki/shared/news/2022-05-24-triangles/community@2x.png)
+
+Players have a chance to earn something too! Once the playlists are closed, whoever has the highest score on each leaderboard will receive a minor prize too:
+
+- **Easy/Normal/Hard/Insane playlist leaders:** 1 month of osu!supporter each
+- **Expert playlist leader:** 1 month of osu! supporter + *triangle aficionado* profile badge
+
+![triangle aficionado profile badge](/wiki/shared/news/2022-05-24-triangles/aficionado@2x.png)
+
+---
+
+Once *everything above* is finalized, we'll announce the results in a **live stream showcase of the finalists** with commentary from some of the judges!
+
+In case one team wins multiple categories, we'll combine their profile badges accordingly. Sorry profile badge collectors, you only receive one.
+
+## Rules
+
+Surprising absolutely nobody, this contest has RULES:
+
+- **Submission must including the following:**
+  - A beatmap spread of *cYsmix - triangles* with **FIVE DIFFICULTIES TOTAL**: Easy, Normal, Hard, Insane, and Expert. [Download the beatmap template here!](https://assets.ppy.sh/artists/2/Songs/cYsmix%20-%20triangles.osz)
+  - A storyboard with all storyboarded content **under ~1 MB total** (including the `.osb` file and difficulty-specific storyboarding).
```

Comment:
is this supposed to imply *storyboarded content only* (minus video, minus hit objects), or the whole mapset folder size? for the latter, it would be clearer to just say "the whole beatmap folder must be under ~1 MB total". the wording is a little suspiciously specific atm

Reply:
woops removed